### DISCUSSION TITLE
Restoring hints, index and integrity ?

### DISCUSSION THREAD

**Discussion by Zerwin** [borgbackup/borg#5983] [Q&A] [answered]
Hi, 

I accidently ran a rm -R over parts of my backups.
Pretty much all of the data is still there, but the hints, index and integrity file got deleted. config is still present. 
Is there any way to restore this backup other than starting from scratch ? 
I tried running borg info on the repository but the command just runs forever without actually showing any info.

**Comment by ThomasWaldmann** [comment, marked as answer]
The first thing you should do is make a backup of whatever you still have in the repo.

After that, try to fix it: run `borg check --repair` on the repo.

If that doesn't help, please give full output of that.

**Reply by Zerwin** [reply]
That worked, thanks
